scRNA-seq-advanced/setup/SCPCP000005-samples/Snakefile

```diff
@@ -0,0 +1,30 @@
+# Workflow for preprocessing SCPCP000005 data used for integration instruction
+# Raw files are expected to be present in the `config[raw_dir]` directory
+
+configfile: "config.yaml"
+outdir = "processed"
```

Comment:
This is the first of a series of suggestions for the input and output. My idea before, poorly expressed, is to use `outdir` as the variable for the base location, but not use the `workdir` setting that snakemake provides, as I have found that to cause more trouble than it is worth.

So here I will get the outdir from your config file. (We could change the name here, but it should work as is. I did run into trouble when I used `basedir` for some reason though)
```suggestion
outdir = config["base_dir"]
```